object_tracker_0/src/inference.py

```diff
@@ -174,7 +175,7 @@ def do_track_objects(predictor, state, bboxes, starting_frame=0, max_frames=10):
     print(f"Adding {len(bboxes)} boxes from tracking from frame {starting_frame} to {starting_frame + max_frames}")
 
     with torch.inference_mode(), torch.autocast("cuda", dtype=torch.float16):
-        for obj_id, bbox in enumerate(bboxes):
+        for obj_id, bbox in bboxes.items():
```

Comment:
what's the difference between enumerate and items?

Reply:
Initially bboxes was a list of coordinates, so enumerate would just return tuples containing the index and the coordinates.  I changed boxes to be a dict that contains { track_id => coordinates }.  Items returns the pair (track_id, coordinates).  That way, we don't lose the track_id (when using enumerate, the first box was always 0, and so on)